src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -199,6 +199,7 @@ export class MoreInfoDialog extends LitElement {
   }
 
   private _computeShowHistoryComponent(entityId) {
+    return true;
```

Comment:
Should probably not be there, leftover from debug?

Reply:
Hehe you think? 😄